core/tools.js

```diff
@@ -1844,6 +1844,39 @@
 				}
 				return acc;
 			}
+		},
+
+		/**
+		 * A set of object helpers.
+		 *
+		 * @property {CKEDITOR.tools.object}
+		 * @member CKEDITOR.tools
+		 */
+		object: {
+			/**
+			 * Return a first key from `obj` which has given value. If key cannot be find, `null` is returned.
```

Comment:
"If key cannot be find, `null` is returned." that part should be explained in `@returns` tag.